Porting/perlgov-team-update

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env perl
+use v5.30.0;
+use warnings;
+use Encode qw(decode);
+use JSON::PP;
+
+my $file = 'Porting/core-team.json';
+open my $fh, '<:encoding(UTF-8)', $file
+  or die "can't read $file: $!\n";
+
+my $json = do { local $/; <$fh> };
+
+my $data = JSON::PP->new->decode($json);
+
+my $pod = q{};
+
+for my $key (qw( active inactive )) {
+  $pod .= qq{=head2 \u$key Members\n\n=over 4\n\n};
+
+  my @items = map {; decode('utf-8', "<$_>", Encode::FB_CROAK) }
```

Comment:
This seems like a double-decode?

Reply:
You're right, that was meant to be an encode, but my mind wandered, thanks!